### PR TITLE
Manage dependencies using the yarn package manager

### PR DESCRIPTION
**Pull request by lumie31**
#### What does this PR do?
- This PR aims to manage project dependencies using the yarn package manager
#### Description of Task to be completed?
- Install dependencies using a package manager to allow for easy update/modification if needed/required
#### How should this be manually tested?
- By running the project and seeing if the new installation does not break anything

@sauloperez Based on our conversation, I was able to install the closest version of the dependencies (i18next, jquery) and I hope it doesn't break anything. The bootstrap version installed is the latest because there was no version stated in the `vendor/` directory.

Also, need your go-ahead on if to ignore the `vendor/` directory by adding it to `.gitignore` file or just completely delete it from the project

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "bootstrap": "^4.3.1",
```
Comment (sauloperez) [inline package.json]: I found the bootstrap version we're using in the header of the `css/bootstrap.css`: `Bootstrap v4.0.0-alpha.6 (https://getbootstrap.com)` so we should be able to use that one https://www.npmjs.com/package/bootstrap/v/4.0.0-alpha.6

Comment (lumie31) [reply package.json]: I've revised this as specified. Cheers!!!

**Comment by lumie31** [comment]
Hello @sauloperez Wanted to check if you're free at the moment or might be free later today, we could have that video call you spoke about

**Comment by sauloperez** [comment]
I was actually thinking about planning it for next week as I have a tight schedule today but we can have a quick chat at 17h CEST. Does that work for you? we share time zone if I'm not mistaken (which is really fortunate :trollface: )

**Comment by lumie31** [comment]
That's 5 PM and I think it's okay with me. Looking forward to the call. What platform are we going to be using for the call?

**Comment by sauloperez** [comment]
Let's meet in https://meet.jit.si/Coopdevs. See you there!

**Pull request merged**